article/app/controllers/LiveBlogController.scala

```diff
@@ -66,16 +66,16 @@ class LiveBlogController(
       val topicList = topMentionsService.getTopics(path)
       page.map(ParseBlockId.fromPageParam) match {
         case Some(ParsedBlockId(id)) =>
-          renderWithRange(path, PageWithBlock(id), filter, topMentions, topicList) // we know the id of a block
+          renderWithRange(path, PageWithBlock(id), filter, topMentions, topicList, topics) // we know the id of a block
```

Comment:
as per this chat, I'm thinking it may be clearer to name the parameter in the url "topics", and in the model (and throughout the code) to more explicitly have two fields
availableTopics (instead of topicList in this case) - to represent all available topics for the current article 
selectedTopics (instead of topics in this case) - to represent the topics selected through the "topics" query parameter

What do you think?

Reply:
I realise this needs to happen more widely and there's another renaming PR but I think it's important to get this right, such that we can both expand on these filters easily in the future and to help others follow what's going on a bit more easily. Sorry to derail.

Reply:
Thanks for the comments Alina! This should be ready for review again now.

Reply:
I think the parameter in the renderArticle endpoint should still be called topics to match the name of the query param

Reply:
good spot!